### PR TITLE
zh: simplify kind tutorial

### PR DESCRIPTION
**Pull request by cofyc**
### What is changed, added or deleted? (Required)

#149

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/deploy-tidb-from-kubernetes-kind.md [hunk 1]
```diff
@@ -17,97 +17,34 @@ kind 通过使用 Docker 容器作为集群节点模拟出一个本地的 Kubern
...
-    > **注意：**
-    >
-    > 通过该脚本启动的 Kubernetes 集群默认有 6 个节点，Kubernetes 版本默认为 v1.12.8，每个节点默认挂载数为 9。你可以通过启动参数去修改这些参数：
-    >
-    > {{< copyable "shell-regular" >}}
-    >
-    > ```shell
-    > hack/kind-cluster-build.sh --nodeNum 2 --k8sVersion v1.14.6 --volumeNum 3
-    > ```
+参考 `kind` [快速使用文档](https://kind.sigs.k8s.io/docs/user/quick-start) 安装
```
Comment (cofyc) [inline zh/deploy-tidb-from-kubernetes-kind.md:27]: people can always refer to official docs for how to install and create a cluster locally.

File: zh/deploy-tidb-from-kubernetes-kind.md [hunk 2]
```diff
@@ -17,97 +17,34 @@ kind 通过使用 Docker 容器作为集群节点模拟出一个本地的 Kubern
...
-- [Docker](https://docs.docker.com/install/)：版本 >= 17.03
-- [Helm Client](https://helm.sh/docs/intro/install/)：版本 >= 2.11.0 并且 < 2.16.4
-- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)：版本 >= 1.12，建议 1.13 或更高版本
-
-    > **注意：**
-    >
-    > 不同 kubectl 版本，输出可能略有不同。
-
-- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/)：版本 >= 0.4.0
-- [net.ipv4.ip_forward](https://linuxconfig.org/how-to-turn-on-off-ip-forwarding-in-linux) 需要被设置为 `1`
```
Comment (cofyc) [inline zh/deploy-tidb-from-kubernetes-kind.md:37]: not necessary

File: zh/deploy-tidb-from-kubernetes-kind.md [hunk 3]
```diff
@@ -17,97 +17,34 @@ kind 通过使用 Docker 容器作为集群节点模拟出一个本地的 Kubern
...
+- [docker](https://docs.docker.com/install/)：版本 >= 17.03
+- [helm](https://helm.sh/docs/intro/install/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+- [kind](https://kind.sigs.k8s.io/)
```
Comment (cofyc) [inline zh/deploy-tidb-from-kubernetes-kind.md:23]: encourage people to install latest version if they don't have these installed

File: zh/deploy-tidb-from-kubernetes-kind.md [hunk 4]
```diff
@@ -17,143 +17,76 @@ kind 通过使用 Docker 容器作为集群节点模拟出一个本地的 Kubern
...
-
-    > **注意：**
-    >
-    > 不同 kubectl 版本，输出可能略有不同。
-
-- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/)：版本 >= 0.4.0
+- [docker](https://docs.docker.com/install/)：版本 >= 17.03
+- [helm](https://helm.sh/docs/intro/install/)：helm 2 或 helm 3 最新稳定版
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)：版本 >= 1.12
+- [kind](https://kind.sigs.k8s.io/)：版本 >= 0.7.0 建议最新版
```
Comment (cofyc) [inline zh/deploy-tidb-from-kubernetes-kind.md:23]: encourage people to use the latest version

File: zh/deploy-tidb-from-kubernetes-kind.md [hunk 5]
```diff
@@ -17,143 +17,76 @@ kind 通过使用 Docker 容器作为集群节点模拟出一个本地的 Kubern
...
-    >
-    > 通过该脚本启动的 Kubernetes 集群默认有 6 个节点，Kubernetes 版本默认为 v1.12.8，每个节点默认挂载数为 9。你可以通过启动参数去修改这些参数：
-    >
-    > {{< copyable "shell-regular" >}}
-    >
-    > ```shell
-    > hack/kind-cluster-build.sh --nodeNum 2 --k8sVersion v1.14.6 --volumeNum 3
-    > ```
+参考 `kind` [快速使用文档](https://kind.sigs.k8s.io/docs/user/quick-start) 安装
+`kind` 并创建一个集群。
```
Comment (ran-huang) [inline zh/deploy-tidb-from-kubernetes-kind.md:29]:
```suggestion
参考 `kind` [快速使用文档](https://kind.sigs.k8s.io/docs/user/quick-start)安装 `kind` 并创建一个集群。
```

File: zh/deploy-tidb-from-kubernetes-kind.md [hunk 6]
```diff
@@ -17,143 +17,76 @@ kind 通过使用 Docker 容器作为集群节点模拟出一个本地的 Kubern
...
 
-3. 集群创建完毕后，执行下列命令将 kubectl 的默认配置文件切换到 `kube-config`，从而连接到该本地 Kubernetes 集群：
+以下以 `0.8.1` 版本为例：
```
Comment (ran-huang) [inline zh/deploy-tidb-from-kubernetes-kind.md:31]:
```suggestion
以下以 0.8.1 版本为例：
```

File: zh/deploy-tidb-from-kubernetes-kind.md [hunk 7]
```diff
@@ -17,143 +17,76 @@ kind 通过使用 Docker 容器作为集群节点模拟出一个本地的 Kubern
...
-    standard (default)   kubernetes.io/host-path        8m29s
-    ```
+```
+kubectl cluster-info 
+```
 
 ## 第 2 步：部署 TiDB Operator
 
-配置 PingCAP 官方 chart 仓库:
+参考 `helm` [安装文档](https://helm.sh/docs/intro/install/) 安装 `helm` 或使用以下命令安装最新 helm 3 稳定版：
```
Comment (ran-huang) [inline zh/deploy-tidb-from-kubernetes-kind.md:47]:
```suggestion
参考 [Helm 安装文档](https://helm.sh/docs/intro/install/) 安装 Helm 或使用以下命令安装最新 Helm 3 稳定版：
```

File: zh/deploy-tidb-from-kubernetes-kind.md [hunk 8]
```diff
@@ -17,143 +17,76 @@ kind 通过使用 Docker 容器作为集群节点模拟出一个本地的 Kubern
...
 {{< copyable "shell-regular" >}}
 
 ```shell
-helm search pingcap -l
+kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.0-rc.3/manifests/crd.yaml && \
+kubectl get crd tidbclusters.pingcap.com
 ```
 
-TiDB Operator 使用 [CRD (Custom Resource Definition)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) 扩展 Kubernetes，所以要使用 TiDB Operator，必须先创建 `TidbCluster` 等各种自定义资源类型：
+安装 TiDB Operator，以下使用 helm 3 为例：
```
Comment (ran-huang) [inline zh/deploy-tidb-from-kubernetes-kind.md:64]:
```suggestion
安装 TiDB Operator，以下使用 Helm 3 为例：
```

File: zh/deploy-tidb-from-kubernetes-kind.md [hunk 9]
```diff
@@ -17,143 +17,76 @@ kind 通过使用 Docker 容器作为集群节点模拟出一个本地的 Kubern
...
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
-kubectl get crd tidbclusters.pingcap.com
+helm repo add pingcap https://charts.pingcap.org/
+kubectl create ns pingcap
+helm install --namespace pingcap tidb-operator pingcap/tidb-operator --version v1.1.0-rc.3
 ```
 
-创建 `TidbCluster` 自定义资源类型后，接下来在 Kubernetes 集群上安装 TiDB Operator。
+若是使用 helm 2，参考[安装 Helm](tidb-toolkit.md#使用-helm) 初始化 helm 后，使用以下命令部署：
```
Comment (ran-huang) [inline zh/deploy-tidb-from-kubernetes-kind.md:74]:
```suggestion
若是使用 Helm 2，参考[安装 Helm](tidb-toolkit.md#使用-helm) 初始化 Helm 后，使用以下命令部署 TiDB Operator：
```

**Comment by ran-huang** [comment]
@cofyc Please resolve the conflicts.

**Pull request merged**